web-common/.eslintrc.cjs

```diff
@@ -0,0 +1,29 @@
+module.exports = {
```

Comment:
we should just have one of these and reference it in each independent directory

Reply:
Good call, just added a commit.